pkg/api/defaults-cloud-controller-manager.go

```diff
@@ -10,14 +10,16 @@ import (
 func (cs *ContainerService) setCloudControllerManagerConfig() {
 	o := cs.Properties.OrchestratorProfile
 	staticCloudControllerManagerConfig := map[string]string{
-		"--allocate-node-cidrs":    strconv.FormatBool(!o.IsAzureCNI()),
-		"--configure-cloud-routes": strconv.FormatBool(o.RequireRouteTable()),
-		"--cloud-provider":         "azure",
-		"--cloud-config":           "/etc/kubernetes/azure.json",
-		"--cluster-cidr":           o.KubernetesConfig.ClusterSubnet,
-		"--kubeconfig":             "/var/lib/kubelet/kubeconfig",
-		"--leader-elect":           "true",
-		"--v":                      "2",
+		"--allocate-node-cidrs":         strconv.FormatBool(!o.IsAzureCNI()),
+		"--configure-cloud-routes":      strconv.FormatBool(o.RequireRouteTable()),
+		"--controllers":                 "*,-cloud-node",
```

Comment:
Does this `--controllers` change work for pre-1.16 clusters using CCM?

Reply:
I tested a few days ago with a 1.15 cluster and didn't see a problem, but let me test again to see if it should be made conditional.

Reply:
The 1.15 cloud-controller-manager container starts up ok with this flag according to Kubernetes but logs seem to halt after an unrelated (not!) error, although the container stays healthy. I'll make this flag conditional for k8s >= 1.16.

Reply:
I made the addition of this argument conditional and added a unit test for that.